src/components/icons/BurgerIcon.tsx

```diff
@@ -0,0 +1,56 @@
+/**!
+ * This file was automatically generated with @twilio-labs/svg-to-react
+ */
+import React from "react";
+import { useUID } from "react-uid";
+
+export interface BurgerIconProps {
+    className?: string;
+    size?: number;
+    color?: string;
+    title?: string;
+    decorative?: boolean;
+}
+
+const BurgerIcon = React.memo(
+    ({
+        title = "Burger Icon",
+        decorative = true,
+        className,
+        color,
+        size,
+    }: BurgerIconProps) => {
+        const titleId = useUID();
+        return (
+            <span
+                style={{ color, width: size, height: size, display: "inline-block" }}
+                className={className}
+            >
+                <svg
+                    role="img"
+                    aria-hidden={decorative}
+                    height="100%"
+                    width="100%"
+                    xmlns="http://www.w3.org/2000/svg"
+                    viewBox="0 0 14 14"
+                    aria-labelledby={titleId}
+                >
+                    {title ? <title id={titleId}>{title}</title> : null}
+                    <g
+                        fill="currentColor"
+                        stroke="#606B85"
```

Comment:
Did we manage to make the icons working on light/dark theme?

Reply:
Yeah, I think so... see the images I put up here https://github.com/mannehedlund/emoji-picker-2.x-plugin